### ISSUE TITLE
it doesn't show the count on the very first load

### ISSUE THREAD

**Issue by benatkin** [maccman/spine.todos#3]
Instead of saying `0 left` it just says `left`. If I go into the web inspector (I'm using Chrome) and delete "Task" from LocalStorage or change its value to the empty string it says just `left` again.

It does properly say `0 left` after adding records and deleting them all. In that case, if I look at the localStorage value in Web Inspector, it shows `[]` as the value.

I looked for a fix briefly but an elegant solution didn't jump out at me. I'm not very familiar with Spine so I thought I'd just report the issue and see what you come up with. I look forward to hearing it!

To sum it up:
- If it's the first page load or I delete `Task` from localStorage, the value for `Task` in localStorage is either non-existent or the empty string, and it says `left`.
- If I add an item and delete it, the value for `Task` in localStorage is `[]` and it says `0 left`.

**Comment by maccman** [comment]
Yes, this is an issues I'm going to try and solve as soon as I get a sec. Thanks for the heads up.
